test/filesystems/blk_filesystem_test.rb

```diff
@@ -88,4 +88,28 @@
       end
     end
   end
+
+  describe "#in_network?" do
```

Comment:
You are right. My point was that I wanted to see everything together (a better name for that would be `test/filesystem_test.rb` or `test/filesystems_test.rb`, I guess). Specially because of the NFS vs others thingie. But feel free to leave it where it is.